src/ec/packages/ec-component-expandable/expandable.html.twig

```diff
@@ -79,7 +79,7 @@
     icon_position: "after",
     extra_classes: 'ecl-expandable__toggle',
     extra_attributes: _button_attributes,
-  }|merge(button) %}
+  }|merge(button) only %}
```

Comment:
it should use `_button`, shouldn't it? we want to avoid using the parameters directly in the template